lib/containers/utils.pm

```diff
@@ -315,4 +315,78 @@ sub reset_container_network_if_needed {
     }
 }
 
+sub test_volumes {
+    my %args = @_;
+    my $runtime = $args{runtime};
+
+    # Create a directory with the sticky-bit set
+    assert_script_run('mkdir -m 1777 test');
```

Comment:
```suggestion
    assert_script_run('mkdir -m 1777 container_volume_test');
```

Let's be a bit more distinctive. Other tests might already have created a directory `test`

Reply:
This will have follow-up changes, which I don't comment to not be unhelpfully verbose

Reply:
Fixed.